test/ReactWrapper-spec.jsx

```diff
@@ -2051,6 +2051,12 @@ describeWithDOM('mount', () => {
       expect(parents.at(1).hasClass('bar')).to.equal(true);
       expect(parents.at(2).hasClass('bax')).to.equal(true);
     });
+
+    it('should work with components in the tree', () => {
+      const Foo = () => (<div className="bar" />);
+      const wrapper = mount(<Foo />);
+      expect(wrapper.find('.bar').parent()).to.equal(wrapper.find('Foo'));
```

Comment:
I'm not convinced this should actually work - `Foo` isn't supposed to be in the tree at all; only what Foo *renders* is.